02_oop_part_1/code/src/Inventory.ts

```diff
@@ -0,0 +1,26 @@
+import {Item} from "./Item";
+import {ItemComparator} from "./ItemComparator";
+
+export class Inventory {
+  private items: Item[];
```

Comment:
@serhii-yaremchuk-epam, this variable has never initialized. If you run `usage.ts` you will get ``` Cannot read property 'push' of undefined```